chapters/arrays.tex

```diff
@@ -1357,6 +1357,9 @@ \subsection{Slice Operation}\label{slice-operation}
   and does not require that \lstinline!size(a[1].m) = size(a[2].m)!.  The number of subscripts on \lstinline!m! must not be greater than the number of array dimension
   for \lstinline!m! (the number can be smaller, in which case the missing trailing indices are assumed to be `\lstinline!:!'), and is only valid if
   \lstinline!size(a[1].m[$\ldots$]) = size(a[2].m[$\ldots$])!.
+\item
+  If \lstinline!...! gives an array of records (e.g. \lstinline!foo(time)!) then \lstinline!(...).m! is interpreted as selecting the \lstinline!m! component in that array of records.
```

Comment:
It would be nice to avoid this sort of use of ellipsis.  Maybe like this?
```suggestion
  When the member access operator is applied to a record array, it is interpreted as constructing an array by selecting the member in each record.
```